### PR TITLE
Feature/sof 6081 clean

### PR DESCRIPTION
**Pull request by azech-hqs**

### CODE REVIEW

File: example/method_units/mathematical/discr/mesh/struct/cartesian.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            "value": -0.5,
+            "units": "angstrom"
+        },
+        "offset.y": {
+            "value": -0.5,
+            "units": "angstrom"
+        },
+        "offset.z": {
+            "value": -1.0,
+            "units": "angstrom"
```
Comment (timurbazhirov) [inline example/method_units/mathematical/discr/mesh/struct/cartesian.json]: These `.{x,y,z}` parameters should not have units

File: example/methods_directory/lin_tetrahedron.json [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+                    "value": 0.0
+                },
+                "nPoints.x": {
+                    "value": 4
+                },
+                "nPoints.y": {
+                    "value": 4
+                },
+                "nPoints.z": {
+                    "value": 4
```
Comment (timurbazhirov) [inline example/methods_directory/lin_tetrahedron.json:56]: Per my comment above - just like in the espresso input we don't have the units

File: schema/methods_directory/lin_tetrahedron.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    "allOf": [
+        {
+            "$ref": "../method.json"
+        }
+    ],
+    "properties": {
+        "units": {
+            "type": "array",
+            "items": {
+                "anyOf": [
```
Comment (timurbazhirov) [inline schema/methods_directory/lin_tetrahedron.json]: should this be `oneOf`?

Comment (azech-hqs) [reply schema/methods_directory/lin_tetrahedron.json]: I just checked on https://www.jsonschemavalidator.net/. They both work here. I'll switch it to `oneOf` also due to this SO post: https://stackoverflow.com/questions/15396251/correct-json-schema-for-an-array-of-items-of-different-type

**Review by timurbazhirov — APPROVED** [review]

**Pull request merged**